climada/entity/tag.py

```diff
@@ -42,7 +44,7 @@ def __init__(self, file_name='', description=''):
         description : str, optional
             description of the data
         """
-        self.file_name = str(file_name)
+        self.file_name = file_name
```

Comment:
```suggestion
        self.file_name = str(file_name)
```
Casting is not completely for the birds here, it's imaginable that someone runs `Tag(file_name=pathobject)` where pathobject really _is_ a `Path` object.